pulsar-client-reactive-adapter/src/main/java/org/apache/pulsar/reactive/client/internal/adapter/AdaptedReactiveMessageSender.java

```diff
@@ -192,11 +192,12 @@ private Mono<MessageId> createMessageMono(MessageSpec<T> messageSpec, Producer<T
 	}
 
 	@Override
-	public Flux<MessageId> sendMessages(Flux<MessageSpec<T>> messageSpecs) {
+	public Flux<MessageId> sendAll(Publisher<MessageSpec<T>> messageSpecs) {
```

Comment:
```suggestion
	public Flux<MessageId> send(Publisher<MessageSpec<T>> messageSpecs) {
```
I'd prefer `send` over `sendAll`.

Reply:
I also prefer `send`. I had put `sendAll` because R2DBC has `saveAll`. But reactor-kafka also uses `send`.
Done